packages/material-ui-lab/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -101,6 +101,12 @@ export interface AutocompleteProps
    * @returns {ReactNode}
    */
   renderTags?: (value: any, state: RenderValueState) => React.ReactNode;
+  /**
+   * Object of titles to display when hovering the arrow or clear buttons.
+   * Options: openPopup, closePopup, clearPopup
+   * Default values: 'Open popup', 'Close popup', 'Clear'
+   */
+  titles?: object;
```

Comment:
We will probably want to detail the TypeScript structure.

Reply:
Are we good with this?
``` javascript
titles?: {
    openPopup?: string,
    closePopup?: string,
    clearPopup?: string,
  };
```

Reply:
Required keys, lile?
`openPopup: string,`

Reply:
Aha that's the correct approach, I fixed it.